mycroft/client/text/__main__.py

```diff
@@ -38,11 +39,17 @@ def custom_except_hook(exctype, value, traceback):
 
 def main():
     # Monitor system logs
-    start_log_monitor("/var/log/mycroft/skills.log")
-    start_log_monitor("/var/log/mycroft/voice.log")
-    # logs when using Debian package   TODO: Unify all
-    start_log_monitor("/var/log/mycroft-skills.log")
-    start_log_monitor("/var/log/mycroft-speech-client.log")
+    config = Configuration.get()
+    if 'log_dir' in config:
+        log_dir = os.path.expanduser(config['log_dir'])
+        start_log_monitor(os.path.join(log_dir, 'skills.log'))
+        start_log_monitor(os.path.join(log_dir, 'voice.log'))
+    else:
+        start_log_monitor("/var/log/mycroft/skills.log")
+        start_log_monitor("/var/log/mycroft/voice.log")
+        # logs when using Debian package   TODO: Unify all
+        start_log_monitor("/var/log/mycroft-skills.log")
```

Comment:
Does the Debian package write to /var/log/mycroft-skills.log anymore?  I don't have a Mark 1 up at the moment to check., but I'm pretty sure it doesn't.   If not, let's remove support for those old logs.

Reply:
You're right, those aren't needed anymore. Mark-1 switched a long time ago.

Reply:
PR updated